http-server/src/main/java/com/facebook/airlift/http/server/HttpServer.java

```diff
@@ -427,9 +431,21 @@ private static ServletContextHandler createServletContext(
         context.addServlet(servletHolder, "/*");
 
         for (Map.Entry<String, Servlet> servlet : servlets.entrySet()) {
-            ServletHolder holder = new ServletHolder(servlet.getValue());
-            holder.setInitParameters(ImmutableMap.copyOf(parameters));
-            context.addServlet(holder, servlet.getKey());
+            if (config.isAuthorizationEnabled()) {
+                checkArgument(authorizer != null, "when authorization is enabled, authorizer implementation must be provided");
+                SecureServlet secureServlet = new SecureServlet(servlet.getValue(),
```

Comment:
nit: put `servlet.getValue()` on the new line. In our codebase we either keep all the arguments on the same line, or put each of the arguments on the new line.